src/ResourceManager/Sql/Commands.Sql/Data Sync/Cmdlet/SetAzureSqlSyncMember.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.DataSync.Model;
+using Microsoft.Azure.Management.Sql.Models;
+
+namespace Microsoft.Azure.Commands.Sql.DataSync.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to update an existing sync member
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRmSqlSyncMember", SupportsShouldProcess = true, 
+        ConfirmImpact = ConfirmImpact.Medium)]
+    [CmdletBinding(DefaultParameterSetName = "All")]
+    public class SetAzureSqlSyncMember : AzureSqlSyncMemberCmdletBase
```

Comment:
You do not need a default parameter binding if there is only one parameter set.

Need an OutputType attribute